pkg/odo/cli/component/push.go

```diff
@@ -51,6 +51,13 @@ var pushCmd = &cobra.Command{
 		}
 		componentName := context.Component(argComponent)
 
+		// if the componentName is blank then there is not active component set
+		if len(componentName) == 0 {
+			log.Error(`No component is set as active.
```

Comment:
Why not use "" and `\n` for break-line instead?

Reply:
Make sense however TBH I feel this is too lengthy as a reported error message from odo end. 

Can it be improvised like
```
No active component is available, to activate a component please use `odo component set`.
```
I am not confident after the or - ```or call this command with component name as and argument.````. This means you are passing the argument and of course it is no more blank. So this is unnecessary IMO. Correct me f I am wrong ?

Reply:
this message was mentioned in the issue as something which was present in odo's older version so I just went with it

Reply:
@mik-dass are you talking about the condition? `len(componentName) == 0`?

Reply:
BTW there is typo in the message ```and argument```. Replaced with ```an argument```

Reply:
@girishramnani No I meant the message itself. It seemed too lengthy and for new-line, please use `\n`

Reply:
@amitkrout resolved it

Reply:
Cool LGTM from myside